app/validators.py

```diff
@@ -0,0 +1,18 @@
+def is_valid_skill(skill):
+    valid_skills = {"BJJ", "Karate", "Judo", "KungFu", "Capoeira", "Boxing", "Taekwondo", "Aikido", "KravMaga",
+                    "MuayThai", "KickBoxing", "Pankration", "Wrestling", "Sambo", "Savate", "Sumo", "Kendo",
+                    "Hapkido", "LutaLivre", "WingChu", "Ninjutsu", "Fencing", "ArmWrestling", "SuckerPunch",
+                    "44Magnum"}
+    return skill in valid_skills
```

Comment:
I'm confused as to how using a set is faster than a list. Wouldn't you still be iterating over every element in the set to check if the skill is in the valid_skills set?

Reply:
I have implemented a hash map for valid_skills instead of set. Would you please check if I did it correctly?